### PR TITLE
fix: apply order to CollectorOptionsSection

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

SSIA

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorOptionsSection.vue [hunk 1]
```diff
@@ -102,11 +102,19 @@ const state = reactive({
...
             label: property.title ?? key,
             disableCopy: !state.collectorOptions[key],
-        }));
+        })).sort((a, b) => {
+            const aIndex = order.indexOf(a.name);
+            const bIndex = order.indexOf(b.name);
+            if (aIndex === -1 && bIndex === -1) return a.name.localeCompare(b.name);
+            if (aIndex === -1) return 1;
+            if (bIndex === -1) return -1;
+            return aIndex - bIndex;
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorOptionsSection.vue:116]: ㅇAㅇ!!!

**Review by yuda110 — APPROVED** [review]
LGTM!!

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorOptionsSection.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorOptionsSection.vue:116]: copilot! haha

**Review by sulmoJ — APPROVED** [review]
LGTM!

**Pull request merged**